live-examples/css-examples/backgrounds-and-borders/background-position-x.html

```diff
@@ -37,7 +37,7 @@
 </section>
 
 <div id="output" class="output large hidden">
-    <section id="default-example" class="display-block">
+    <section id="default-example">
```

Comment:
This example, and the other two background-position examples, don't seem to work for me any more: I don't see the star or the background colour. It seems to be the removal of `class="display-block"` that's done this. Do you see that as well?

Reply:
Yes, sorry about that.